pkg/kubernetes/apis/loadtest/v1/spec.go

```diff
@@ -27,9 +33,16 @@ func BuildLoadTestSpec(loadTestType LoadTestType, overwrite bool, distributedPod
 		return lt, ErrRequireTestFile
 	}
 
+	for _, value := range tags {
+		if len(value) > maxTagLength {
```

Comment:
Should we allow empty tags? Or would it be better to filter them out?

Reply:
The input is validated at the request level (not empty, not longer than 63 chars) so I don't think we need to check the empty again coz k8s accepts empty label.